eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -216,6 +216,7 @@ stages:
                           PRBranchName: increment-package-version-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
                           CommitMsg: "Increment package version after release of ${{ artifact.name }}"
                           PRTitle: "Increment version for ${{ parameters.ServiceDirectory }} releases"
+                          CloseAfterOpenForTesting: $(TestPipeline)
```

Comment:
Don't we need this for other places like the docs PR's as well?

Reply:
That has been added at https://github.com/Azure/azure-sdk-for-net/blob/81e43256c59927240501e9f219134c6f5ddf263b/eng/common/pipelines/templates/steps/create-pull-request.yml#L89